platform/ios/CHANGELOG.md

```diff
@@ -2,20 +2,20 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
-## master
-
-* Fixed a crash caused by incorrect `MGLMapViewImpl` renderable size. ([#14810](https://github.com/mapbox/mapbox-gl-native/pull/14810))
-* Fixed queryRenderedFeatues bug caused by incorrect sort feature index calculation. ([#14884](https://github.com/mapbox/mapbox-gl-native/pull/14884))
-
 ## 5.1.0
 
 ### Styles and rendering
 
 * Setting `MGLMapView.contentInset` now moves the map’s focal point to the center of the content frame after insetting. ([#14664](https://github.com/mapbox/mapbox-gl-native/pull/14664))
+* Fixed a rendering performance regression when rendering polylines. ([#14851](https://github.com/mapbox/mapbox-gl-native/pull/14851))
+* Fixed a rendering performance regression introduced in 4.11.0 ([#14907](https://github.com/mapbox/mapbox-gl-native/pull/14907))
+* Fixed an issue that caused cropping symbols under opaque fill layers. ([#14839](https://github.com/mapbox/mapbox-gl-native/pull/14839))
 
 ### Other changes
 
 * The `-[MGLMapView setCamera:withDuration:animationTimingFunction:edgePadding:completionHandler:]` method now adds the current value of the `MGLMapView.contentInset` property to the `edgePadding` parameter. ([#14813](https://github.com/mapbox/mapbox-gl-native/pull/14813))
+* Fixed queryRenderedFeatues bug caused by incorrect sort feature index calculation. ([#14884](https://github.com/mapbox/mapbox-gl-native/pull/14884))
+* Fixed a crash caused by incorrect `MGLMapViewImpl` renderable size. ([#14810](https://github.com/mapbox/mapbox-gl-native/pull/14810))
```

Comment:
This fixed an issue only present in `ios-v5.1.0` prereleases, so we don’t want to include it in this changelog.

For the GitHub release notes about changes in `ios-v5.1.0-beta.1`, though: `MGLMapViewImpl` isn’t public API, so we shouldn’t mention it by name.

> Fixed a crash caused by incorrect renderable size.

Reply:
Added to the GitHub release notes.